### PR TITLE
Fix metric alert grammar for 1 minute

### PR DESCRIPTION
**Pull request by ceorourke**
This fixes the text of a metric alert so that if something happened in the last minute, it will say "... in the last 1 minute" instead of "... in the last 1 minutes".

### CODE REVIEW

File: src/sentry/integrations/metric_alerts.py [hunk 1]
```diff
@@ -74,7 +74,9 @@ def incident_attachment_info(incident, metric_value=None, action=None, method=No
     else:
         metric_and_agg_text = f"{metric_value} {agg_text}"
 
-    text = f"{metric_and_agg_text} in the last {time_window} minutes"
+    interval = "minute" if time_window == 1 else "minutes"
```
Comment (evanpurkhiser) [inline src/sentry/integrations/metric_alerts.py:77]: 🤔 should we be using the translation functions on these things?

Comment (ceorourke) [reply src/sentry/integrations/metric_alerts.py:77]: Is that this `ugettext_lazy` ?

Comment (evanpurkhiser) [reply src/sentry/integrations/metric_alerts.py:78]: Yeah, there's a `_` version of it too

File: src/sentry/integrations/metric_alerts.py [hunk 2]
```diff
@@ -74,7 +75,9 @@ def incident_attachment_info(incident, metric_value=None, action=None, method=No
...
+
+    text = _(f"{metric_and_agg_text} in the last {time_window} {interval}")
```
Comment (evanpurkhiser) [inline src/sentry/integrations/metric_alerts.py:80]: I don't think you can use ugettext like this, it needs to be templatized.

Comment (ceorourke) [reply src/sentry/integrations/metric_alerts.py:80]: :( okay let me try again, idk why but the test I wanna use this on is just freaking out locally so I was hoping CI would catch stuff.

**Review by mgaeta — APPROVED** [review]
